src/Derivative.cpp

```diff
@@ -553,36 +553,6 @@ void ReverseAccumulationVisitor::propagate_adjoints(
         current_func = func;
 
         FuncKey func_key{func.name(), func.num_update_definitions() - 1};
-        // Set up boundary condition for the last adjoint, for
-        // non overwriting scans, we delay the boundary condition
-        // setup since the gradients depend on itself.
-        auto add_boundary_condition = [&](const FuncKey &func_key) {
-            Func &adjoint_func = adjoint_funcs[func_key];
-            const Box &bounds = func_bounds[func.name()];
-            // Save a pointer to the unbounded def. Useful for scheduling
-            FuncKey unbounded_func_key{func.name() + "_unbounded", func_key.second};
```

Comment:
If we are no longer saving/using the "_unbounded" def, shouldn't we also remove the public API for accessing it?

Reply:
Fixed.